model/project.go

```diff
@@ -138,7 +138,7 @@ type BuildVariantTaskUnit struct {
 	// If Activate is set to false, then we don't initially activate the task.
 	Activate *bool `yaml:"activate,omitempty" bson:"activate,omitempty"`
 	// Group is set if an inline task group is defined on the build variant.
-	Group *TaskGroup `yaml:"group,omitempty" bson:"group,omitempty"`
+	Group *TaskGroup `yaml:"task_group,omitempty" bson:"task_group,omitempty"`
```

Comment:
Sorry I wasn't clear but preferably the variable name would match the tags, i.e. TaskGroup.

Reply:
Ahh I see, apologies fixed now